appoptics_test/server_mux_test.go

```diff
@@ -27,6 +27,7 @@ func teardown() {
 }
 
 func TestMain(m *testing.M) {
+	defer server.Close()
```

Comment:
Can we remove the `teardown` function if we're doing this instead?

Reply:
thought a bit more and just keeping `teardown`